src/containers/Home/Search/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react'
+import "./style.css"
+import axios from 'axios';
+import City from '../../Type/city';
+import Job from '../../Type/job';
+function Search() {
+    const [jobs, setJob] = useState<Job[]>();
+    const [city, setCity] = useState<City[]>();
+    useEffect(()=>{
+       axios.get(`${process.env.REACT_APP_API_URL}/products`)
+       .then(res=>{
+        setJob(res.data)
+       })
+       .catch(error => console.log(error));
+
+       axios.get(`${process.env.REACT_APP_API_URL}/qltt`)
+       .then(res=>{
+        setCity(res.data);
+       })
+       .catch(error => console.log(error));
+    },[])
+    const RenderLocation = ():JSX.Element =>{
```

Comment:
This should use as HTML element no need to create a Component here

Reply:
i've changed